images/installer/Dockerfile

```diff
@@ -1,51 +1,44 @@
-# Using playbook2image as a base
-# See https://github.com/openshift/playbook2image for details on the image
-# including documentation for the settings/env vars referenced below
-FROM registry.centos.org/openshift/playbook2image:latest
+FROM centos:7
 
 MAINTAINER OpenShift Team <dev@lists.openshift.redhat.com>
 
+USER root
+
```

Comment:
Is there any reason to move the `USER` and `RUN` commands before the `LABEL`s? If not, wouldn't it be better to leave the `LABEL`s first to keep the metadata type of content at the top? The caching related benefits of that are probably not relevant in our case, but still wondering if it would make sense from a file organization POV?